gsy_e_sdk/setups/Grid_Operator_API_scripts/redis_aziiz_strategy.py

```diff
@@ -0,0 +1,138 @@
+# flake8: noqa
+# pylint: disable=duplicate-code
+"""
+Template file to implement Aziiz grid fees
+strategy through the gsy-e-sdk api client using Redis.
+"""
+import os
+import json
+from time import sleep
+from gsy_e_sdk.redis_aggregator import RedisAggregator
+from gsy_e_sdk.redis_market import RedisMarketClient
+from gsy_e_sdk.utils import log_grid_fees_information
+
+current_dir = os.path.dirname(__file__)
```

Comment:
I find `current_dir' a bit misleading, because you are determining the path of the module and not the current directory that you are in. How about renaming it to `module_dir` or `script_dir`